src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -667,5 +664,5 @@ protected void finishLocalSetup() throws CheckstyleException {
             }
         }
     }
-
 }
+
```

Comment:
Please confirm this. Is this a second empty line? All Java files should have an empty line at the bottom already.

Reply:
fixed this too.